### DISCUSSION TITLE
No immich-machine-learning openvino image tagged for v1.112.1?

### DISCUSSION THREAD

**Discussion by mpfl** [immich-app/immich#11969] [Q&A] [answered]
I'm using Docker on Debian Bullseye and want to upgrade from v1.111.0 to v1.112.1 but it looks like there isn't an immich-machine-learning openvino image tagged for that version.

The [docker-compose.yml file](https://github.com/immich-app/immich/releases/download/v1.112.1/docker-compose.yml) provided for [the v1.112.1 release](https://github.com/immich-app/immich/releases/tag/v1.112.1) still says:

```
    # For hardware acceleration, add one of -[armnn, cuda, openvino] to the image tag.
```

I can pull v1.112.1 for immich-server and immich-machine-learning, but can't pull v1.112.1-openvino for immich-machine-learning:

```
$ docker pull ghcr.io/immich-app/immich-server:v1.112.1
v1.112.1: Pulling from immich-app/immich-server
efc2b5ad9eec: Already exists
7f05e058a5ff: Already exists
a98359e09be4: Already exists
ae1e854ca0d6: Already exists
88e291100e5a: Already exists
00e776823f77: Already exists
47fe8fabe8db: Already exists
7abab93e41c7: Already exists
ffdb2cf435d0: Already exists
2d49993e6870: Already exists
f3e4e92ddfea: Already exists
89fb1e8bfe34: Already exists
4f4fb700ef54: Already exists
8a24d226c801: Pull complete
1604b6d44e4a: Pull complete
26fba743c3d7: Pull complete
d98185ca3ed6: Pull complete
d5947531d5eb: Pull complete
9368a4f778fe: Pull complete
24189da93706: Pull complete
5fcb2ae21c1b: Pull complete
69f8e42c9dc9: Pull complete
9d966614834c: Pull complete
eb13a47ca499: Pull complete
Digest: sha256:c4e817f0eadbd9a6c2699cc884d5e7070428daec813c17db77d31fcca5b78ca6
Status: Downloaded newer image for ghcr.io/immich-app/immich-server:v1.112.1
ghcr.io/immich-app/immich-server:v1.112.1

$ docker pull ghcr.io/immich-app/immich-machine-learning:v1.112.1-openvino
Error response from daemon: manifest unknown

$ docker pull ghcr.io/immich-app/immich-machine-learning:v1.112.1
v1.112.1: Pulling from immich-app/immich-machine-learning
e4fff0779e6d: Already exists
d97016d0706d: Already exists
53db1713e5d9: Already exists
a8cd795d9ccb: Already exists
de3ba92de392: Already exists
093aa5632067: Pull complete
4156e82d817d: Pull complete
57a70e9c23a8: Pull complete
83d9d7bc8ba8: Pull complete
2cafb2e50108: Pull complete
035bc3bbaa47: Pull complete
f1eea2d16c2e: Pull complete
Digest: sha256:9600eff5a66ae426293f00b171711bc1647c85cf966d759ee08ab2d05e0580b5
Status: Downloaded newer image for ghcr.io/immich-app/immich-machine-learning:v1.112.1
ghcr.io/immich-app/immich-machine-learning:v1.112.1


```

Is there an undocumented change to the immich-machine-learning image?

**Comment by alextran1502** [comment, marked as answer]
Correct, you can use the tag `v1.112.0` for this. There was some dependencies issues for `v1.112.1` for it to get built but there is no changes to the image content

**Reply by mpfl** [reply]
Thanks for the swift response.

So I use `immich-machine-learning:v1.112.0-openvino` with `immich-server:v1.112.1`?

If so, the provided `docker-compose.yml` won't work any more because it requires both images to be tagged with the same version - stored in `IMMICH_VERSION`.

I'm comfortable with modifying the `docker-compose.yml` myself, but I'm mindful that users less familiar with how docker compose works might struggle.

**Reply by alextran1502** [reply]
Correct, that should be fine
